src/westpa/core/binning/mab.py

```diff
@@ -243,6 +246,18 @@ def map_mab(coords, mask, output, *args, **kwargs):
         # output is the main list that, for each segment, holds the bin assignment
         output[i] = holder
 
+    if bin_log and report:
+        if westpa.rc.sim_manager.n_iter:
+            with open(expandvars("$WEST_SIM_ROOT/binbounds.log"), 'a') as bb_file:
+                bb_file.write(f'{westpa.rc.sim_manager.n_iter}\n')  # Iteration Number
+                for n in range(ndim):
+                    bb_file.write(f'{np.linspace(minlist[n], maxlist[n], nbins_per_dim[n] + 1)}\t')  # Write binbounds per dim
```

Comment:
Could we store the bin boundaries above in a variable and then write out that variable here instead of recreating what would be the boundaries?

Sorry, I know the PR is merged and I just got time to check it out now.

Reply:
Because of how we [loop through each segment and then each dimensions](https://github.com/westpa/westpa/pull/357/files/1787e58291dc43754fb608ae5957cac90618d038#diff-90eeb2361b949db85d2024ec96a3bc1add2c8d0c2865d5c56866e996b74461f0L148-L157), I was getting duplicate outputs if I tried to save it beforehand. 

We could theoretically add in extra checks, but it's just cleaner this way when we calculate it one last time.